electron-cash

```diff
@@ -26,93 +26,128 @@
 # ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+# pylint: disable=C0103
+# pylint: enable=C0103
+"""
+Electron Cash - lightweight Bitcoin Cash client
+"""
+import getpass
 import os
 import sys
+import threading
+
+import electroncash.web as web
+from electroncash import (
+    Network,
+    SimpleConfig,
+    daemon,
+    keystore,
+    networks,
+    util,
+    websockets,
```

Comment:
The `websockets` module relies on an optionl module that isn't always available: `SimpleWebSocketServer`.  It should not always be unconditionally imported because this will cause the app to no longer run for the majority of users that don't even care about websockets.  Please revert this back to a lazy import for only the "websocket_server" code path as we had before.

Reply:
Thank you, I reverted this line.